newIDE/app/src/AssetStore/AssetStoreContext.js

```diff
@@ -247,7 +247,7 @@ export const AssetStoreStateProvider = ({
 
       const timeoutId = setTimeout(() => {
         console.info('Pre-fetching assets from asset store...');
-        fetchAssetsAndFilters();
+        fetchAssetsAndFilters(false);
```

Comment:
Here you have an example of the "bad implicitness": you pass false, so it means that other components must consider that by default the environment was the production one (which is fair, but in the future you could want to load staging assets first, and this should need only a change in a single place.)